### PR TITLE
feat(colors): Update the global color palette

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

This brings the colors more in line with xrpl.org.  This greatly simplifies the color system and setups a redesign of how we represent different transaction types.

Notable changes:

- Remove the horizontal rule on the homepage
- Update the background colors of ledger hash and corresponding validations #229
- Transaction and account flags are purple instead of blue
- Transaction table header rows are now black and have a hover state

### Context of Change

The new color guide.

Screenshot 2023-01-09 at 2 07 51 PM

### Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking change that only restructures code)

##### Before
Before - Homepage

##### After
After - Homepage

##### Before
Before - Account

##### After
After - Account

##### Before
Before - Nodes

##### After
After - Nodes

##### Before
Before - Table Hover

##### After
After - Table Hover

##### Before
Before - Token

##### After
After - Token

##### Before
Before - Transaction

##### After
After - Transaction

##### Before
Before - Upgrade Status

##### After
After - Upgrade Status

##### Before
Before - Validators

##### After
After - Validators

###

###

## Future Tasks

The colors for transactions will be changed in a future PR.

### CODE REVIEW

**Comment by jonathanlei** [comment]
is this still a draft?

**Comment by ckniffen** [comment]
> is this still a draft?

Yeah it is.  Still working with design on a few edge cases and I got to take screen shots.

File: src/containers/shared/css/tooltip.scss [hunk 1]
```diff
@@ -36,39 +36,20 @@
...
-    letter-spacing: 0px;
+    color: $black-40;
+    letter-spacing: 0;
   }
 
   .unl {
     color: $green;
   }
 
-  .parent {
```
Comment (ckniffen) [inline src/containers/shared/css/tooltip.scss:47]: These are all unused

File: src/containers/shared/css/simpleTab.scss [hunk 2]
```diff
@@ -59,25 +59,19 @@ $index-width: 324px;
...
       padding: 15px 0;
-      border-bottom: 1px solid $gray-800;
+      border-bottom: 1px solid $black-70;
       gap: 0 5px;
 
       @include for-size(tablet-portrait-up) {
         padding: 15px 15px 15px 0;
       }
 
-      &.empty {
```
Comment (ckniffen) [inline src/containers/shared/css/simpleTab.scss:69]: Now unused.

**Review by JST5000 — APPROVED** [review]
LGTM

**Review by jonathanlei — APPROVED** [review]

**Pull request merged**